saas_app/requirements.txt

```diff
@@ -1,6 +1,100 @@
 asgiref==3.7.2
 aws-sns-message-validator==0.0.5
 awscli==1.28.0
+azure-appconfiguration==1.1.1
```

Comment:
Do we need all these dependencies?  I think we would probably only need loganalytics and the common ones? Might be a pain to widdle down though :)